hw/_index.md

```diff
@@ -42,14 +42,14 @@ Finally, we provide the same set of information for all available [top level des
 * [Earl Grey Chip Datasheet]({{< relref "hw/top_earlgrey/doc" >}})
 * [Earl Grey Chip Specification]({{< relref "hw/top_earlgrey/doc/design" >}})
 * [Earl Grey Chip DV Document]({{< relref "hw/top_earlgrey/doc/dv" >}})
-* [DV simulation results, with coverage (nightly)](https://reports.opentitan.org/hw/top_earlgrey/dv/latest/results.html)
-* [Connectivity results (nightly)](https://reports.opentitan.org/hw/top_earlgrey/conn/jaspergold/latest/results.html)
-* [AscentLint results (nightly)](https://reports.opentitan.org/hw/top_earlgrey/lint/ascentlint/latest/results.html)
-* [Verilator lint results (nightly)](https://reports.opentitan.org/hw/top_earlgrey/lint/verilator/latest/results.html)
-* [Style lint results (nightly)](https://reports.opentitan.org/hw/top_earlgrey/lint/veriblelint/latest/results.html)
-* [DV Style lint results (nightly)](https://reports.opentitan.org/hw/top_earlgrey/dv/lint/veriblelint/latest/results.html)
-* [Synthesis results (nightly)](https://reports.opentitan.org/hw/top_earlgrey/syn/latest/results.html)
-* [CDC results (nightly)](https://reports.opentitan.org/hw/top_earlgrey/cdc/latest/results.html)
+* [Chip-level DV simulation results, with coverage (nightly)](https://reports.opentitan.org/hw/top_earlgrey/dv/latest/report.html)
+* [Chip-level Connectivity results (nightly)](https://reports.opentitan.org/hw/top_earlgrey/conn/jaspergold/latest/report.html)
+* [Chip-level AscentLint results (nightly)](https://reports.opentitan.org/hw/chip_earlgrey_asic/lint/ascentlint/latest/report.html)
```

Comment:
the `chip_earlgrey_asic` hierarchy has been used before already - we just did not point these links to the correct place.

Reply:
so we have multiple "chip-level" instances at this hierarchy level:
`top_earlgrey`
`chip_earlgrey_asic`
`chip_earlgrey_verilator`
`chip_earlgrey_cw310`

for the first one I agree - this is already done.
for the others we need a different hierarchy to push the reports to (this has already been done like this, but the dashboard was still pointing at `top_earlgrey` instead of `chip_earlgrey_asic`).

I agree that it would be best to align the report structure with the repository tree - but the problem is that the toplevel structure would have to be changed quite significantly in order to achieve this.

one example could be to restructure as follows:
```
└── hw
    └── tops
        ├── earlgrey
        │   ├── ip
        │   │   └── ast
        │   └── ip_autogen
        │       ├── top_earlgrey
        │       ├── top_earlgrey_asic
        │       ├── top_earlgrey_cw310
        │       └── top_earlgrey_verilator
        └── englishbreakfast
...
```

Reply:
CC @tjaychen: did you discuss any folder hierarchy changes with Chris for the new multi-top repo strategy?

Reply:
Do we really need to run fpga / verilator tops through lint?

Reply:
we currently do not lint `top_earlgrey_verilator`. for FPGA it would be useful, but we keep it in a separate config right now (so that it can be run as a one-off pass locally). as for `chip_earlgrey_asic`, that one is definitely needed.

what we could do is remove `top_earlgrey` and only run `chip_earlgrey_asic`, since it includes `top_earlgrey`. that way we could at least work around this issue for the time being.

Reply:
I implemented this workaround so that all chip-level results share the same path stem (`hw/top_earlgrey/`).
we may want to discuss the top-level hierarchy in the future, however, to see whether we can improve the structure.